styles/globals.css

```diff
@@ -93,13 +93,12 @@ h2, h3 {
 
 
 
-/* ensuring a 100% height page */
 html,
 body,
 body > div:first-child,
 div#__next,
 div#__next > div {
-  height: 100%;
+  background-color: #ffffff;
```

Comment:
Body background is black without this, it's noticeable when you pull the page up/down... actually this might only be on "prefers dark mode" 🤔

Reply:
It may be a `prefers` issue, it may just be that the height of this body element runs on out of height and the default background color of unstyled html elements is black.